src/lib/util/helpers.ts

```diff
@@ -4,7 +4,7 @@ export function isPresent<T>(o: T | undefined | null): o is T {
 
 export function batchesOf<T>(array: T[], count: number): T[][] {
   const tmp = [...array];
-  const batches = [];
+  const batches: Array<any> = [];
```

Comment:
Can we use `T[]` as a type for `batches`?

Reply:
done